runtime/shared_common/CacheMap.cpp

```diff
@@ -1775,21 +1774,8 @@ SH_CacheMap::runEntryPointChecks(J9VMThread* currentThread, void* address, const
 			}
 		}
 	}
-	
-	if (!hasClassSegmentMutex) {
-		if (acquireClassSegmentMutex) {
-			Trc_SHR_Assert_False(_ccHead->hasWriteMutex(currentThread));
-			enterLocalMutex(currentThread, currentThread->javaVM->classMemorySegments->segmentMutex, "class segment mutex", "CM runEntryPointChecks");
-			classSegmentMutexAcquired = true;
-			hasClassSegmentMutex = true;
-		}
-	}
-	itemsAdded = refreshHashtables(currentThread, hasClassSegmentMutex);
-	if (classSegmentMutexAcquired) {
-		exitLocalMutex(currentThread, currentThread->javaVM->classMemorySegments->segmentMutex, "class segment mutex", "CM runEntryPointChecks");
-	}
 
-	if (-1 == itemsAdded) {
+	if ((itemsAdded = refreshHashtables(currentThread, hasClassSegmentMutex)) == -1) {
```

Comment:
Since the code is modified, the constant here and the next change should be on the left.

Reply:
Fixed.